soh/include/z64player.h

```diff
@@ -612,6 +620,8 @@ typedef struct Player {
     /* 0x0A86 */ s8         unk_A86;
     /* 0x0A87 */ u8         unk_A87;
     /* 0x0A88 */ Vec3f      unk_A88; // previous body part 0 position
-} Player; // size = 0xA94
+    /* 0x0A94 */ FlagType   pendingFlagType; // type of flag to set when Player_SetPendingFlag is called
+    /* 0x0A96 */ s32        pendingFlag; // which flag to set when Player_SetPendingFlag is called
```

Comment:
maybe making these a struct would be good, then player could just have `pendingFlag`, and that struct could have the flag id and flag type

Reply:
I like that idea, didn't think of it for some reason. Done.